test/components/Modals/UploadDocumentModal.test.jsx

```diff
@@ -24,3 +25,21 @@ it('renders cancel/upload group as column mobile', () => {
 
   expect(cssProperty.flexDirection).toBe('column');
 });
+
+it('renders initial state and elements correctly', () => {
+  const { getByText, getByRole, getByLabelText } = render(<MockUploadDocumentModal />);
+
+  // Check for initial state values
+  const modalTitle = getByText('Upload Document');
```

Comment:
would this not be `const modalTitle = getByRole('heading', {name: whatever const you initialize earlier for it});` ?